### PR TITLE
Update Makefile to release and build for multiple platforms

### PR DESCRIPTION
**Pull request by luxas**
@DirectXMan12 PTAL

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,41 @@
-all: build
+REGISTRY?=directxman
```
Comment (DirectXMan12) [inline Makefile]: I'm `directxman12` on dockerhub, too

File: Makefile [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+endif
+ifeq ($(ARCH),arm)
+        BASEIMAGE?=armhf/busybox
+endif
+ifeq ($(ARCH),arm64)
+        BASEIMAGE?=aarch64/busybox
+endif
+
+all: build
 build:
```
Comment (DirectXMan12) [inline Makefile]: I'd prefer to have the `build` target just be for normal `go building`, and have a `cross-build` target or something like that for this.

Comment (luxas) [inline Makefile]: Actually this builds things for just one arch where amd64 is the default. However, I'll split it to `build` and `docker-build`

**Comment by DirectXMan12** [comment]
I think this looks fine.  I *might* change `all` to just be `build` after the fact (haven't decided yet), but otherwise LGTM :+1:.  Thanks!

**Pull request merged**